website/assets/styles/03-elements/links.scss

```
@@ -9,7 +9,7 @@ a:link:not([class]),
 a:visited:not([class]),
 .e-link {
   color: currentColor;
-  text-decoration: dotted underline;
+  text-decoration: underline;
```

```suggestion
  text-decoration: underline;
  text-decoration-style: dotted;
```

It does, just probably not shorthand without a prefix. So let's write it longhand.

---

Amazing, thanks!